os/kernel/task_monitor/task_monitor_utils.c

```diff
@@ -21,9 +21,9 @@
 #include <sys/prctl.h>
 #include <sys/types.h>
 
-void task_monitor_register(void)
+void task_monitor_register(int time)
```

Comment:
1. I think time is not good. How about using interval?
2. If you want to change the param, please change the prototype also.